docs/development/setup-advanced.md

```diff
@@ -58,51 +58,70 @@ ignoring the parts about `vagrant` (since you're not using it).
 ## Installing directly on Windows 10 with WSL 2
 
 Zulip's development environment is most easily set up on Windows using
-the [WSL 2](https://docs.microsoft.com/en-us/windows/wsl/wsl2-about)
+the Windows Subsystem for Linux [(WSL 2)]
+(https://docs.microsoft.com/en-us/windows/wsl/wsl2-about)
 installation method described here.
 
-1. Install WSL 2 by following the instructions provided by Microsoft
-   [here](https://docs.microsoft.com/en-us/windows/wsl/install).
+1. Enable virtualization through your BIOS settings.
+   This sequence depends on your specific hardware and brand, but here
+   are [some basic instructions.]
+   (https://www.thewindowsclub.com/disable-hardware-virtualization-in-windows-10)
 
-1. Install the `Ubuntu 18.04` Linux distribution from the Microsoft
-   Store.
+1. Install WSL 2 by following the instructions provided by Microsoft
+   [here](https://docs.microsoft.com/en-us/windows/wsl/setup/environment).
 
-1. Launch the `Ubuntu 18.04` shell and run the following commands:
+1. Launch the Ubuntu shell as an administrator and run the following command:
```

Comment:
I like making this less specific in a way that inevitably gets stale (the Ubuntu release number), and also referring to upstream instructions for how to install WSL.